master/usage/configuration/ip-in-ip.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Configuring IP-in-IP
+---
+
+If your network fabric performs source/destination address checks 
+and drops traffic when those addresses are not recognized, it may be necessary to
+use IP-in-IP encapsulation of the inter-workload traffic.
+  
+This is often the case for public-cloud environments where you have limited control
+over the fabric and you are not peering your Calico BGP network with the rest of the
+network.
+
+Calico can be configured to use IP-in-IP encapsulation by enabling the IPIP option
+on the [IP pool resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool). 
+When enabled, Calico will use IP-in-IP encapsulation when routing packets *to*
+workload IPs falling in the IP pool range.
```

Comment:
Yeah I guess this is becoming an IP Pool configuration doc more than IP-in-IP....  and then there is also the External connectivity doc.